templates/telecommunications/index.html

```diff
@@ -0,0 +1,208 @@
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>Why choose Canonical?</h2>
+    <ul class="p-list is-split">
+      <li class="p-list__item is-ticked">Keep <abbr title="Capital expenditure">CAPEX</abbr> and <abbr title="Operating expenses">OPEX</abbr> costs down with virtualisation and automation</li>
+      <li class="p-list__item is-ticked">Peace of mind with a pre-tested solution</li>
+      <li class="p-list__item is-ticked">Get started quicker with our speedy onboarding process</li>
+      <li class="p-list__item is-ticked">High-availability <abbr title="ervice-level agreement">SLA</abbr>s aligned with accepted industry practices</li>
+      <li class="p-list__item is-ticked">Stay ahead with easy upgrades between OpenStack versions and fast access to the latest OpenStack features with 100% upstream integration</li>
+    </ul>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>The complete cloud automation toolset</h2>
+      <p>Operational discipline &mdash; built on automation and repeatability &mdash; has long been crucial to the profitability of CSPs. Canonical applies the same principles to the way clouds are built and run.  From design to deployment and operations, every aspect is managed with Canonical&rsquo;s proven automation toolset, including provisioning tool MAAS, management tool Landscape, and application modeling tool Juju.</p>
+      <ul class="p-list">
+        <li><a href="/cloud/juju">Find out more about Juju&nbsp;&rsaquo;</a></li>
+        <li><a href="/server/maas">Find out more about MAAS&nbsp;&rsaquo;</a></li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>NFVI cloud build</h2>
+      <p>Canonical&rsquo;s Foundation Cloud Build for Telcos is a pre-tested, pre-integrated OpenStack cloud built using a proven reference architecture and tailored to the needs of CSPs. Think of it as a fixed price, NFV-ready cloud, deployed by Canonical on your premises in just a few weeks.</p>
+      <p><a href="/cloud/foundation-cloud">Find out more<span class="u-off-screen"> about NFVI cloud build</span>&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>A telco-grade managed cloud</h2>
+      <p>A number of telcos rely on Canonical&rsquo;s fully managed cloud service. Get Telco Grade SLAs and predictable operational costs for your OpenStack cloud, on your servers at your locations. We build, operate, scale and upgrade your cloud, while you concentrate on building your applications in flexible and agile environments.</p>
+      <p><a href="/cloud/managed-cloud">Find out more<span class="u-off-screen"> about Canonical&rsquo;s Foundation Cloud Build</span>&nbsp;&rsaquo;</a></p>
```

Comment:
Isn't this link to Canonical's managed cloud. Not Foundation Cloud Build?

Reply:
Done